File: algorithms/sorting/bubble sort/bubbleSort.js
const fs = require("fs");
const values = fs
  .readFileSync("values.txt", "utf8")
  .split(",")
  .map((i) => Number(i));

function bubbleSort(values) {
  for (let i = 0; i < values.length; i++) {
    for (let j = 0; j < values.length - i - 1; j++) {
      if (values[j] > values[j + 1]) {
        const temp = values[j];
        values[j] = values[j + 1];
        values[j + 1] = temp;
      }
    }
  }
  console.log(values);
}

bubbleSort(values);

File: algorithms/sorting/bubble sort/bubble_sort.py
data = open("values.txt", "r")
values = [int(v) for v in data.read().split(",")]

def bubble_sort(values):
    for i in range(len(values)):
        for j in range(len(values)-i-1):
            if values[j] > values[j+1]:
                values[j], values[j+1] = values[j+1], values[j]
    print(values)

bubble_sort(values)